libraries/testkit/syncgateway.py

```diff
@@ -349,6 +410,45 @@ def __repr__(self):
         return "SyncGateway: {}:{}\n".format(self.hostname, self.ip)
 
 
+def get_buckets_from_sync_gateway_config(sync_gateway_config_path):
```

Comment:
we are redefining it again. Can we have one common method and use it in all files?

Reply:
changed the reference to this file and removed the definition from cluster.py.